config/locales/en.yml

```diff
@@ -15,6 +15,132 @@ en:
     success: The payment was successful.
   all_rights_reserved: All rights reserved.
   back: Go back
+  receipt:
```

Comment:
I just got realizing, we probably should namespace all of the translations under `pay` shouldn't we? That way we don't conflict with user specified translations for something simple like `back` that they may use in other places.

No need to do that in this PR, but just a thought I had.